package.json

```diff
@@ -57,6 +57,8 @@
     "eslint-plugin-jsdoc": "^39.6.2",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-prettier": "^4.2.1",
+    "html-inline-script-webpack-plugin": "^3.1.0",
+    "html-webpack-plugin": "^5.5.0",
```

Comment:
Should these just be added to the `snaps-execution-environments` package.json?

Reply:
Oops, I intended to only add these to `snaps-execution-environments`. Will fix.